package.json

```diff
@@ -31,7 +31,7 @@
   },
   "dependencies": {
     "abort-controller": "^3.0.0",
-    "backfill": "^6.1.11",
+    "backfill": "^6.1.12",
```

Comment:
I think this should be `^6.1.13`? https://github.com/microsoft/backfill/commit/6b5c826da02e461afbfbac2f166138c6d09922a9

Reply:
hmmm i might not have bumped to latest-latest